module/src/components/progressBar/progressBar.hooks.ts

```diff
@@ -67,37 +67,37 @@ export const useProgress = ({ trickle, trickleAmount, trickleInterval, maximum,
     [set]
   );
 
-  const onTimeout = React.useCallback(() => {
+  const onTimeout = React.useCallback(async () => {
     if (trickle) {
       increment(Math.min((100 - progressRef.current) * 0.05, trickleAmount || 2));
 
-      // eslint-disable-next-line @typescript-eslint/no-use-before-define
+      // eslint-disable-next-line @typescript-eslint/no-use-before-define, @typescript-eslint/no-floating-promises
       setInterval();
```

Comment:
you can do `void setInterval()` to avoid having to ignore the no floating promises error